util/_dump-redis.sh

```
@@ -1,7 +1,52 @@
+VALID_ENVS=("dev" "stg" "prd")
+VALID_MODES=("default" "cache" "assessment")
+
+ENV=${1:-}
+if [[ -z "$ENV" ]]; then
+  echo "Usage: $0 <env> [mode]"
+  echo "  env  = one of: ${VALID_ENVS[*]}"
+  echo "  mode = one of: ${VALID_MODES[*]} (default: default)"
+  exit 1
+fi
+
+if [[ ! " ${VALID_ENVS[*]} " =~ " ${ENV} " ]]; then
```

Shellcheck (0.11.0)

[warning] Remove quotes from right-hand side of =~ to match as a regex rather than literally.

(SC2076)